<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1274</width>
    <height>865</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Matthew Yien Yelp Like App</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="businessTab">
       <attribute name="title">
        <string>Businesses</string>
       </attribute>
       <widget class="QGroupBox" name="businessGroup">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>230</y>
          <width>1231</width>
          <height>561</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(152, 255, 152);
font: 10pt &quot;Arial&quot;;
</string>
        </property>
        <property name="title">
         <string>Filter Categories</string>
        </property>
        <widget class="QTableWidget" name="businessTable">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>50</y>
           <width>921</width>
           <height>191</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Select Category</string>
         </property>
        </widget>
        <widget class="QListWidget" name="categoryList">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>221</width>
           <height>191</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>30</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Businesses</string>
         </property>
        </widget>
        <widget class="QPushButton" name="refreshButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>280</y>
           <width>51</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Reload</string>
         </property>
        </widget>
        <widget class="QGroupBox" name="popularBusinessesGroup">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>250</y>
           <width>1151</width>
           <height>301</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="title">
          <string>Popular Businesses (In Zipcode for Selected Category)</string>
         </property>
         <widget class="QTableWidget" name="popularBusinessesTable">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>21</y>
            <width>1131</width>
            <height>271</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
         </widget>
        </widget>
        <widget class="QPushButton" name="searchButton">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>80</y>
           <width>51</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
        <widget class="QPushButton" name="clearButton">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>160</y>
           <width>51</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="businessLocation">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>10</y>
          <width>491</width>
          <height>221</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(68, 158, 157);
font: 10pt &quot;MS Shell Dlg 2&quot;;
</string>
        </property>
        <property name="title">
         <string>Select Location</string>
        </property>
        <widget class="QComboBox" name="stateList">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Arial&quot;;</string>
         </property>
         <property name="text">
          <string>State</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>20</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Arial&quot;;</string>
         </property>
         <property name="text">
          <string>City</string>
         </property>
        </widget>
        <widget class="QListWidget" name="cityList">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>40</y>
           <width>181</width>
           <height>171</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>20</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Arial&quot;;</string>
         </property>
         <property name="text">
          <string>Zipcode</string>
         </property>
        </widget>
        <widget class="QListWidget" name="zipList">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>40</y>
           <width>151</width>
           <height>171</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="businessLocation_2">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>10</y>
          <width>741</width>
          <height>221</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(203, 255, 250);
font: 10pt &quot;Arial&quot;;</string>
        </property>
        <property name="title">
         <string>Zipcode Statistics</string>
        </property>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Arial&quot;;</string>
         </property>
         <property name="text">
          <string># of Businesses</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>20</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Arial&quot;;</string>
         </property>
         <property name="text">
          <string>Top Categories</string>
         </property>
        </widget>
        <widget class="QTableWidget" name="topCategory">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>40</y>
           <width>601</width>
           <height>171</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
</string>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
        <widget class="QListWidget" name="numBusinesses">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
